script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -1,23 +1,23 @@
 <Page1>
 I wake up absentmindedly.
-The arm reaching out to the glasses feels slightly heavy.
-I put on my glasses, wondering if it's a slight fever like it was someone else's business.
+My arm reaching out to my glasses feels slightly heavy.
+I put on my glasses, wondering if I had a slight fever like it was someone else's business.
 <Page2>
 Even as my vision returns to normal, I still feel a slight dizziness.
-It resembles the symptoms of a cold, but this is more like a morning fever.
-It's like stuck engine heated on a flame which gets alleviated simply through the passage of time.
+It may resemble the symptoms of a cold, but this is more like a morning fever.
+It's just like a halted engine being warmed up by a fire, it'll subside with time.
 <Page3>
-There is a knock on the door.
+There's a knock on the door.
 After a short pause―――
 <Page4>
-"Pardon the intrusion."
-With a single bow, Hisui enters the room.
+"Pardon my intrusion."
+With a bow, Hisui enters the room.
 <Page5>
 "Good morning, Hisui. Thanks for coming as always."
 "Good morning, Shiki-sama. I see that you are already awake."
 <Page6>
 "Yeah. I slept very well last night. I'll go to the kitchen as soon as I get dressed.
-...Tell Kohaku-san that I have a slight fever, so she should give me a lower portion today."
+...Tell Kohaku-san that I have a slight fever, so she should give me a smaller portion today."
```

Comment:
technically it's more like "I'd like you to let Kohaku-san..." but that feels a little forceful for this exchange in English. @rschlaikjer @yk1512 how do you guys feel? For now I'd promote it to a question and phrase it like this:
>……琥珀さんにはちょっと熱があるから、量は少なめだって伝えてほしい」 
...Can you let Kohaku-san know that since I have a small fever, I'd like a smaller portion today?